lib/utils.js

```diff
@@ -214,7 +214,7 @@ export function buildSQLQuery (base, payload = {}, operators = [], betweens = []
  *
  */
 export function isGeoPayloadOK (payload) {
-  const split = payload.coordinates.split(',');
+  const split = isArray(payload.coordinates) ? payload.coordinates : payload.coordinates.split(',');
```

Comment:
What's the need for changing this? Does `coordinates` come in as an array from the Hapi parser?